content/docs/command-reference/run.md

```diff
@@ -421,9 +421,9 @@ Define a stage with both regular dependencies as well as parameter dependencies:
 
 ```dvc
 $ dvc run -n train \
-          -d matrix-train.p -d train_model.py -o model.p \
+          -d train_model.py -d matrix-train.p  -o model.p \
           -p seed,train.lr,train.epochs
-          python train_model.py matrix-train.p model.p
+          python train_model.py 20200105 model.p
```

Comment:
where does `20200105` come from?

Reply:
It's yesterdays' date. Added for consistency because in the previous uses of this same sample command (in https://dvc.org/doc/command-reference/run#example-overwrite-an-existing-stage) a date is passed.

Reply:
p.s. now that I see it I realize maybe it's not a date but anyway, it's just an 8-digit number.